text/0039-reduce-module-size.md

```diff
@@ -0,0 +1,533 @@
+# Reduce aws-cdk-lib package size
+
+* **Original Author(s):**: @madeline-k
+* **Tracking Issue**: #39
+* **API Bar Raiser**: tbd
+
+The AWS CDK v2 framework library, `aws-cdk-lib` contains all packages from AWS
+CDK v1 in one monolithic package. While this is a great value for customers in
+making dependency management much easier, it has also resulted in increasing the
+unpacked npm package size to 234 MB, at the time of writing. This feature
+reduces the size of the package to under 100 MB.
+
+## Working Backwards
+
+* **CHANGELOG**:
+
+feat(aws-cdk-lib): Reduce aws-cdk-lib package size.
+
+‼️ If you use any of the following Constructs in a network-restricted
+environment, you might encounter problems upgrading to this version. ‼️
+
+* lambda_layer_kubectl.KubectlLayer
+* aws_eks.KubectlProvider
+* aws_eks.HelmChart
+* aws_eks.KubernetesManifest
+* aws_eks.KubernetesObjectValue
+* aws_eks.KubernetesPatch
+* lambda_layer_awscli.AwsCliLayer
+* aws_s3_deployment.BucketDeploymentß
+* aws_stepfunctions_tasks.EmrContainersStartJobRun
+* lambda_layer_node_proxy_agent.NodeProxyAgentLayer
+* aws_eks.FargateProfile
+* aws_eks.Cluster
+
+Please see [#1234](https://github.com/aws/aws-cdk/issues/1234) for more details,
```

Comment:
I guess this is meant as a placeholder, but this actually links to an unrelated issue right now 😛.

Maybe use #1234567 instead?